commercetools-models/src/test/java/io/sphere/sdk/categories/CategoryFixtures.java

```diff
@@ -50,6 +52,32 @@ public static void withCategory(final BlockingSphereClient client, final Supplie
         }
     }
 
+    public static void withCategories(final BlockingSphereClient client, final List<Supplier<? extends CategoryDraft>> creator, final Consumer<List<Category>> user) {
+
+        List<Category> categories = creator.stream().map(singleCreator -> {
+            final CategoryDraft categoryDraft = singleCreator.get();
+            final String slug = englishSlugOf(categoryDraft);
+            final PagedQueryResult<Category> pagedQueryResult = client.executeBlocking(CategoryQuery.of().bySlug(Locale.ENGLISH, slug));
+            pagedQueryResult.head().ifPresent(
+                    category -> client.executeBlocking(CategoryDeleteCommand.of(category))
+            );
+            final Category category = client.executeBlocking(CategoryCreateCommand.of(categoryDraft));
+            LOGGER.debug(() -> "created category " + category.getSlug() + " id: " + category.getId());
```

Comment:
I prefer tests without log output, because you can run our tests in IDEAs debugger. What do you think?